src/components/Table/Table.tsx

```diff
@@ -160,9 +163,9 @@ export const Table = <T extends Record<string, any>>({
 			data,
 			manualPagination: overridePagination,
 			defaultColumn: {
-				maxWidth: 300,
+				maxWidth: 800,
 				minWidth: 30,
-				width: "auto",
+				width: 150,
```

Comment:
wouldn't this make more sense to default to "auto"? 
```suggestion
				width: resizableColumns ? 150 : "auto",
```

Reply:
It has to be a number.